.env.example

```
@@ -48,9 +42,9 @@ ENDPOINT_PASSWORD=endpoint
 # OAuth
 # ===========================================
 OAUTH_GOOGLE_CLIENT_ID=
-OAUTH_GOOGLE_SECRET_ID=
+OAUTH_GOOGLE_CLIENT_SECRET=
 OAUTH_GITHUB_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OAUTH_GITHUB_CLIENT_ID key should go before the OAUTH_GOOGLE_CLIENT_ID key

(UnorderedKey)